docs/website/docs/plus/getting-started/advanced_tutorial.md

```diff
@@ -8,276 +8,87 @@ import Link from '../../_plus_admonition.md';
+`dlt+` provides tools to help you package a project for distribution.  
+This makes your project installable via `pip` and easier to share across your organization.
+
+To create a packaged project, add the `--package` option when initializing:
 
 ```sh
 dlt project init arrow duckdb --package my_dlt_project
 ```
 
-This will provide you with the same project as used in the basic tutorial but places in a module directory named `my_dlt_project` and provided with a simple PEP compatible `pyproject.toml` file. You will also receive a default package `__init__.py` file to make the module accssible from the outside when installed:
+This creates the same basic project as in the tutorial, but places it inside a module named `my_dlt_project`, and includes a basic `pyproject.toml` file following PEP standards. 
+You’ll also get a default `__init__.py` file to make the package usable after installation:
 
 ```sh
 .
-├── my_dlt_project/       # your project module
-│   ├── __init__.py       # your package entry point
-│   ├── dlt.yml           # the dlt+ project manifest
-│   └── ...               # more project files
+├── my_dlt_project/       # Your project module
+│   ├── __init__.py       # Package entry point
+│   ├── dlt.yml           # dlt+ project manifest
+│   └── ...               # Other project files
 ├── .gitignore
 └── pyproject.toml        # the main project manifest
 ```
 
-Your `dlt.yml` will look and work exactly the same as in the non-packaged version. The only difference is the module structure and the presence of the `pyproject.toml` file. The pyproject.toml has a special entry point setting that makes the dlt+ project discoverable by dlt+:
+Your `dlt.yml` works exactly the same as in non-packaged projects. 
+The key difference is the module structure and the presence of the `pyproject.toml` file. 
+The file includes a special entry point setting to let dlt+ discover your project:
 
 ```toml
 [project.entry-points.dlt_package]
 dlt-project = "my_project"
 ```
 
-You can still run the pipeline the same way as before with the cli commands from the root folder:
+You can still run the pipeline as usual with the cli commands from the root folder:
 
 ```sh
 dlt pipeline my_pipeline run
 ```
 
-If you look at the module `__init__.py` file that was created for you, you can see the full interface of your packaged project that the user of your package will have access to. It is very similar to the interface of the `current` interface of the flat project, there is an additional feature that selects the `access` profile by default. It is up to you to adapt this `__init__.py` file to your needs.
+If you open the `__init__.py` file inside your project module, you'll see the full interface that users of your package will interact with.
+This interface is very similar to the `current` interface used in flat (non-packaged) projects. The main difference is that it automatically uses the `access` profile by default.
```

Comment:
Can you please link to the documentation on the "current" interface - let me know if you need help figuring out how to link there